src/credentialStore.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+"use strict";
+import * as keytarType from "keytar";
+import * as vscode from "vscode";
+import { Constants } from "./constants";
+
+export class CredentialStore {
+    public static getPassword(account: string): Promise<string> {
+        try {
+            return this.keytar.getPassword(Constants.ExtensionId, account);

```

Comment:
If keytar could not get the value of input key, its null or throw error?

Reply:
It's null

Reply:
error is only thrown when keystar is null?

Reply:
error is only thrown when

- keystar is null
- OS does not have a credential store

Reply:
OS does not have a credential store will make keystar null right?

Reply:
I don't think so.  keystar is just a npm package. If there is no credential store, keystar could still exist. But when there is no credential store, this.keytar.getPassword() will throw error.